README.md

```diff
@@ -218,18 +218,34 @@ open-attestation deploy token-registry "My Sample Token" MST --network ropsten
 ✔  success   Token registry deployed at 0x4B127b8d5e53872d403ce43414afeb1db67B1842
 ```
 
+#### Deploy new title escrow
+
+Deploys a title escrow contract on the blockchain
+
+```bash
+open-attestation deploy title-escrow --network <NETWORK> --address <TOKEN_REGISTRY_ADDRESS> --beneficiary <BENEFICIARY_ADDRESS> --holder <HOLDER_ADDRESS>
+```
+
+Example - with private key set in `OA_PRIVATE_KEY` environment variable (recommended). [More options](#providing-the-wallet).
+
+```bash
+open-attestation deploy title-escrow --network ropsten --address 0x4B127b8d5e53872d403ce43414afeb1db67B1842 --beneficiary 0x6FFeD6E6591b808130a9b248fEA32101b5220eca --holder 0x6FFeD6E6591b808130a9b248fEA32101b5220eca
+
+✔  success   Title escrow deployed at 0xB26B4941941C51a4885E5B7D3A1B861E54405f90
+```
+
 #### Issue document to token registry
 
-Issue a hash to a token registry deployed on the blockchain
+`Issue` a hash to a token registry deployed on the blockchain. The `tokenId` option would be used to indicate the document hash, and the `to` option to indicate the title escrow address the document is mapped to.
 
 ```bash
-open-attestation token-registry issue --address <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> --to <TO> [options]
+open-attestation token-registry issue --network <NETWORK> --address <TOKEN_REGISTRY_ADDRESS> --tokenId <TOKEN_ID> --to <TO> [options]
```

Comment:
if im not wrong need `0x` prefix for TOKEN_ID <- https://www.openattestation.com/docs/transferable-record/issuing-transferable-record#issuing-transferable-records